x/evm/keeper/call_evm_test.go

```diff
@@ -76,13 +76,13 @@ func (suite *KeeperTestSuite) TestCallEVMWithData() {
 			true,
 		},
 		{
-			"fail empty data",
+			"pass with empty data",
 			types.ModuleAddress,
 			func() []byte {
 				return []byte{}
 			},
 			false,
-			false,
+			true,
```

Comment:
Because previously the precompile used for WEVMOS was the erc20 one, which does not implement any fallback. Now the werc20 precompile manage this case and the tx is not failing.

Reply:
ok makes sense, thank you